site-src/implementation-table.md

```diff
@@ -0,0 +1,25 @@
+| envoyproxy     | :x:                                         | :white_check_mark:            | :white_check_mark:        | :white_check_mark:                    | :white_check_mark:      | :white_check_mark:        | :white_check_mark:      | :white_check_mark:     | :white_check_mark:     | :white_check_mark:       | :white_check_mark:                | :white_check_mark:        | :white_check_mark:        | :x:                      |
+| istio          | :x:                                         | :white_check_mark:            | :white_check_mark:        | :white_check_mark:                    | :white_check_mark:      | :white_check_mark:        | :white_check_mark:      | :white_check_mark:     | :white_check_mark:     | :white_check_mark:       | :white_check_mark:                | :white_check_mark:        | :white_check_mark:        | :x:                      |
+| kumahq         | :x:                                         | :white_check_mark:            | :white_check_mark:        | :white_check_mark:                    | :white_check_mark:      | :white_check_mark:        | :white_check_mark:      | :white_check_mark:     | :white_check_mark:     | :white_check_mark:       | :x:                               | :x:                       | :x:                       | :x:                      |
+| nginxinc       | :x:                                         | :white_check_mark:            | :white_check_mark:        | :x:                                   | :white_check_mark:      | :white_check_mark:        | :x:                     | :white_check_mark:     | :white_check_mark:     | :x:                      | :x:                               | :x:                       | :x:                       | :x:                      |
+| projectcontour | :x:                                         | :white_check_mark:            | :white_check_mark:        | :white_check_mark:                    | :white_check_mark:      | :white_check_mark:        | :white_check_mark:      | :white_check_mark:     | :white_check_mark:     | :white_check_mark:       | :white_check_mark:                | :white_check_mark:        | :white_check_mark:        | :x:                      |
+| solo.io        | :x:                                         | :white_check_mark:            | :white_check_mark:        | :white_check_mark:                    | :white_check_mark:      | :white_check_mark:        | :white_check_mark:      | :x:                    | :x:                    | :x:                      | :x:                               | :x:                       | :x:                       | :x:                      |
```

Comment:
@xtineskim I think we might need to show both "Organization" and "Project" here for now. Otherwise we end up with incomplete information. For example, in this case, most people would want to know the product name ("Gloo Gateway"), while in the case of Kong above, the "Kubernetes Ingress Controller" product name is really not sufficient. Until we have a better solution in conformance reports, I think we'll just have to settle for reporting both.

@mlavacca Do you think it's worth adding a new field (or merging fields) in conformance reports to make this easier to handle? Maybe we should just have a single product name field that represents everything?

Reply:
in the reports we already have `organization` and `project`, which if merged provide a unique tuple. Acutally, this is very related to my comment above: https://github.com/kubernetes-sigs/gateway-api/pull/2874#pullrequestreview-2003274625. I agree we should address both org+project and implementation version.

Reply:
gotcha, i'll add them in 👍